requirements.txt

```diff
@@ -0,0 +1,4 @@
+click
```

Comment:
How are these requirements decided? What about other imports like `koji` or `logging`?

Reply:
I'm actually dropping that file for now. It was there for use with Travis, but that's going to require more work than I initially thought. I'll split that into a separate PR.